pkg/config/vault.go

```diff
@@ -19,6 +19,8 @@ var (
 		"vaultBasePath",
 		"vaultPipelineName",
 		"vaultPath",
+		"skipVault",
+		"noOverwrite",
```

Comment:
I think we should make it a bit more self explaining.
```suggestion
		"vaultDisableOverwrite",
```

Reply:
Better, but also to be added to the documentation.